docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
+
+###### It is recommended to have minimum m5.xlarge setup.
+
+The images for the openshift-logging need to be pulled from the quay,
+sync the images with the internal registry and update the cluster with the new images for containers.
+
+
+Note: Steps are to be followed till OCP 4.2 is GAed, These steps can be skipped after that
+
+1. Create a quay.io account [https://quay.io]
```

Comment:
please use syntax for link and make it like:
Create an account to [quay.io](https://quay.io).

Reply:
> @Shrivaibavi I would make this documentation part of #837 to have everything in one place. WDYT?

@petr-balogh When we are in the idea of moving the logging-deployment to ocs-ci, isn't it much easier for people to follow the doc that's placed in /docs folder ?

Reply:
I am not telling of move this somewhere else, I meant just have all this documentation as part of PR #837 to have all in one place, so ppl when will do the review will see the documentation as well.  Because if you will merge this one earlier than PR  #837  you will have inconsistency that the documentation is not valid yet.

Reply:
Done